infrastructure/zk/src/docker.ts

```diff
@@ -72,7 +75,7 @@ export async function restart(container: string) {
 }
 
 export async function pull() {
-    await utils.spawn('docker-compose pull postgres rskj dev-liquidity-token-watcher dev-ticker tesseracts elastic');
+    await utils.spawn('docker-compose pull postgres rskj dev-ticker tesseracts elastic');
```

Comment:
Could we please add a comment here to specify that we need to restore this change once we publish the image?

Reply:
done